src/libraries/System.Private.CoreLib/src/System/ReadOnlySpan.cs

```diff
@@ -217,6 +217,11 @@ public bool IsEmpty
         /// </summary>
         public static ReadOnlySpan<T> Empty => default;
 
+        public static ReadOnlySpan<T> CastUp<TDerived>(ReadOnlySpan<TDerived> items) where TDerived : class, T
```

Comment:
I think this (and in the ref) should be:
```suggestion
        public static ReadOnlySpan<T> CastUp<TDerived>(ReadOnlySpan<TDerived> items) where TDerived : class?, T
```